third_party/tensorflow_patches/disable_forced_mkl.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/third_party/mkl/build_defs.bzl b/third_party/mkl/build_defs.bzl
+index 806b157bad6..6c9f0172b97 100644
+--- a/third_party/mkl/build_defs.bzl
++++ b/third_party/mkl/build_defs.bzl
+@@ -33,8 +33,9 @@ def if_mkl(if_true, if_false = []):
+     """
+     return select({
+         "@org_tensorflow//third_party/mkl:build_with_mkl_aarch64": if_true,
+-        "@org_tensorflow//tensorflow:linux_x86_64": if_true,
+-        "@org_tensorflow//tensorflow:windows": if_true,
```

Comment:
This patch looks like its also disabling `mkl` for linux, not only for windows. Is that correct? (Anything that speeds up these builds sounds good to me)

Reply:
Yes, it disables it for both unless you request it through the relevant Bazel flags.

Reply:
Or at least, that's the intent. I haven't actually haven't tested on Linux whether this succeeds — if you run
```
bazel cquery "somepath(:build_pip_pkg, @mkl_dnn_v1//:mkl_dnn)" --define=build_with_mkl=false --define=enable_mkl=false --define=tensorflow_mkldnn_contraction_kernel=0
```
and get something that says "Empty query results" then MKL is successfully disabled.